site/using.md

```diff
@@ -196,3 +196,82 @@ Commit pushed: 708b63a
 SERVICE             STATUS   UPDATES
 default/helloworld  success  
 ```
+
+# Recording user and message with the triggered action
+
+Issuing a deployment change results in a version control change/git commit, keeping the
+history of the actions. Flux daemon can be started with several flags, that impact the commit
+information:
+
+---------------------------------------------------------------------------------------
+flag              | purpose                       | default
+---------------------------------------------------------------------------------------
+git-user          | committer name                | Weave Flux
+---------------------------------------------------------------------------------------
+git-email         | committer name                | support@weave.works
+---------------------------------------------------------------------------------------
+set-git-author    | override of the commit author | false
+---------------------------------------------------------------------------------------
+
+Actions triggered by a user through the Weave Cloud UI or the CLI `fluxctl`
+tool, can have the commit author information customized. This is handy for providing extra context in the
+notifications and history. Whether the customization is possible, depends on the Flux daemon (fluxd)
+`set-git-author` flag. If set, the commit author will be customized in the following way:
+
+## Actions triggered through Weave Cloud
+
+Weave Cloud UI sends user parameter, value of which is the username (email) of the user logged into
+Weave Cloud.
+
+## Actions triggered through `fluxctl`
+
+`fluxctl` provides the following flags for the author adjustment:
```

Comment:
Being uncharacteristically pedantic, I would say `--message` isn't author adjustment.

Reply:
You use the term "commit customization" below -- that would work here as well I reckon.

Reply:
Done